Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -40,7 +40,7 @@ python3 create-external-cluster-resources.py --rbd-data-pool-name <pool_name> --
 * `--rgw-tls-cert-path`: (optional) RADOS Gateway endpoint TLS certificate file path
 * `--rgw-skip-tls`: (optional) Ignore TLS certification validation when a self-signed certificate is provided (NOT RECOMMENDED)
```

Comment:
edit the link for 
`Run the python script [create-external-cluster-resources.py](htt`

Reply:
to make it use the external folder